player/lua/stats.lua

```diff
@@ -736,29 +741,31 @@ local function add_video(s)
     if ro and (r["w"] ~= ro["dw"] or r["h"] ~= ro["dh"]) then
         append_resolution(s, ro, "Output Resolution:", "dw", "dh")
     end
-    append_resolution(s, {w=scaled_width, h=scaled_height}, "Scaled Resolution:")
+    local scale = nil
     if not mp.get_property_native("fullscreen") then
-        append_property(s, "current-window-scale", {prefix="Window Scale:"})
+        scale = mp.get_property_native("current-window-scale")
     end
+    append_resolution(s, {w=scaled_width, h=scaled_height, s=scale}, "Scaled Resolution:")
+    append_property(s, "deinterlace", {prefix="Deinterlacing:"})
```

Comment:
should be hidden if not deinterlacing IMO

Reply:
Fair, done.